boms/cloud-oss-bom/RELEASING.md

```diff
@@ -34,6 +34,10 @@ new release is available on Maven Central.
     * https://github.com/googleapis/google-cloud-java/blob/master/TROUBLESHOOTING.md
 * Merge the dependabot PR that updates libraries-bom in https://github.com/GoogleCloudPlatform/java-docs-samples/pulls
 * Manually edit and update any pom.xml files in https://github.com/GoogleCloudPlatform/java-docs-samples that dependabot missed
+    * Go to go/java-live
+    * Sort by title
+    * Scroll down until you find the section with titles "chore(deps): update dependency com.google.cloud:libraries-bom to v<version>"
```

Comment:
I see 104 PRs as of now...

Reply:
Jeff and Stephanie are working on some ways to automate more of this.